metrics2/metrics2/src/test/java/io/helidon/metrics/MetricsSupportTest.java

```diff
@@ -144,4 +151,21 @@ void testBaseMetricsDisabled() {
         assertFalse(myBase.getGauges().containsKey(METRIC_USED_HEAP), "Base registry incorrectly contains "
                 + METRIC_USED_HEAP + " when base was configured as disabled");
     }
+
+    @Test
+    void testPrometheusDataNoTypeDups() throws Exception {
+        Set<String> found = new HashSet<>();
+        String data = MetricsSupport.toPrometheusData(app, base);
+        try (BufferedReader reader = new BufferedReader(new StringReader(data))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] tokens = line.split(" ");
+                if (tokens.length > 3 && tokens[1].equals("TYPE")) {
+                    String metric = tokens[2];
+                    assertFalse(found.contains(metric));
+                    found.add(metric);
```

Comment:
How about combining these into `assertTrue(found.add(metric))`?

Because performance is _so_ important inside the test!

Reply:
Makes it less readable IMO